### DISCUSSION TITLE
Unexpected token in JSON at position 0

### DISCUSSION THREAD

**Discussion by sergeidyga** [postmanlabs/postman-flows#190] [Q&A] [answered]
Unable to parse JSON in flows. Error message:
"Unexpected token .. in JSON at position 0"

Example flow to reproduce the issue.
1. Send Request: curl https://api.github.com/users/defunkt
2. Parse Json: $.`body`.`login`

<img width="816" alt="Screenshot 2022-01-03 at 12 07 39" src="https://user-images.githubusercontent.com/87765162/147919113-61fcdd59-564f-4dac-bfdb-59d558377f99.png">

**Comment by samay-v** [comment, marked as answer]
Hey, @sergeidyga 
This is because the data inside of `body.login` is not in JSON format. 
Can you help me understand what your usecsae here is?

**Reply by sergeidyga** [reply]
Hi @samay-v!  I simply try to get `login` value (e.g `curl -s https://api.github.com/users/defunkt | jq .login`) and pass it down the flow

**Reply by samay-v** [reply]
Got it. You can achieve this without using the `parse JSON` block. For example, If you want to use it in a variable in the next request, simply connect a request to the response port and assign the value like such: 

![image](https://user-images.githubusercontent.com/28357106/148200726-d924fde9-7d10-4667-aa0e-3f4733a7db5f.png)
![image](https://user-images.githubusercontent.com/28357106/148200725-00267d2d-859f-408b-9ea3-9119ead5a084.png)

Here I am adding a condition to check if the login field is null or not:

![image](https://user-images.githubusercontent.com/28357106/148201186-a2de511d-22cb-4e10-8255-f42b3442f1f4.png)
